cmd/registry/cmd/apply/testdata/lifecycle.yaml

```diff
@@ -6,32 +6,31 @@ data:
   displayName: Lifecycle
   description: A series of stages that an API typically moves through in its lifetime
   stages:
-  - id: concept
-    displayName: Concept
-    description: Description of the business case and user needs for why an API should
-      exist
-    displayOrder: 0
-  - id: design
-    displayName: Design
-    description: Definition of the interface details and proposal of the API contract
-    displayOrder: 1
-  - id: develop
-    displayName: Develop
-    description: Implementation of the service and its API
-    displayOrder: 2
-  - id: preview
-    displayName: Preview
-    description: Staging of implementations in the pre-production phase
-    displayOrder: 3
-  - id: production
-    displayName: Production
-    description: API available for production workloads
-    displayOrder: 4
-  - id: deprecated
-    displayName: Deprecated
-    description: API not recommended for new consumers
-    displayOrder: 5
-  - id: retired
-    displayName: Retired
-    description: API no longer available for use
-    displayOrder: 6
+    - id: concept
```

Comment:
Notice the change to dictionary indentation - this is really the only reason we had to regenerate this YAML test data. (The default indent width also changed to 4 but we set it back to 2 with the custom `marshalYAML()` in `patch/yaml.go`)